README.md

```diff
@@ -1,54 +1,203 @@
+	* [GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos)
+* [Building](https://github.com/TASVideos/BizHawk/blob/master/README.md#building)
+* [Usage](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage)
+	* [TASing](https://github.com/TASVideos/BizHawk/blob/master/README.md#tasing)
+	* [Cores](https://github.com/TASVideos/BizHawk/blob/master/README.md#cores)
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+	* [Testing](https://github.com/TASVideos/BizHawk/blob/master/README.md#testing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+## Features and systems
 
-## Supported Systems
+The BizHawk common features (across all cores) are:
+* format, region, and integrity detection for game images
+* 10 save slots with hotkeys and ∞ named savestates
+* speed control, including frame stepping and rewinding
+* memory view/search/edit in all emulated hardware components
+* input recording (making TAS movies)
+* screenshotting and recording audio + video to file
+* firmware management
+* input, framerate, and more in a HUD over the game
+* emulated controllers via a comprehensive input mapper
+* Lua control over core and frontend (Windows only)
+* hotkey bindings to control the UI
 
- * Nintendo Entertainment System / Famicom / Famicom Disk System (NES/FDS)
- * Super Nintendo (SNES)
- * Nintendo 64
- * Game Boy, Game Boy Color, and Super Game Boy
- * Game Boy Advance
- * Sony PlayStation
- * Sega Master System, Game Gear, and SG-1000
- * Sega Genesis / Sega-CD
- * Sega Saturn
- * PC-Engine (TurboGrafx-16) / CD-ROM & SuperGrafx 
- * Atari 2600
- * Atari 7800
- * Atari Lynx
- * ColecoVision
- * TI-83 Calculator
- * Wonderswan and Wonderswan Color 
- * Apple II
- * Commodore 64
- * Sinclair ZX Spectrum
+Supported consoles and computers:
 
-## Resources
+* N64 and [all](http://tasvideos.org/Bizhawk/N64.html) peripherals; Playstation (PSX); Saturn; Virtual Boy
+* Game Boy Advance; Game Boy Color; Neo Geo Pocket (Color); WonderSwan (Color)
+* Genesis and [all](https://bitbucket.org/eke/genesis-plus-gx/src/b573cd25853f9f8b5b941fc36506835e228144c6/wiki/Features.md?at=master&fileviewer=file-view-default) peripherals; SNES; TurboGrafx / SuperGrafx
+* Atari Lynx; Game Boy; Game Gear; TI-83
+* Atari 7800; Commodore 64 and [some](http://tasvideos.org/Bizhawk/C64.html) peripherals; Master System; NES; ZX Spectrum and [some](http://tasvideos.org/Bizhawk/ZXSpectrum.html) peripherals
+* Apple II; Atari 2600; ColecoVision; IntelliVision
+* [More](http://tasvideos.org/Bizhawk/CoreRoadMap.html) coming soon..?
```

Comment:
On what principle do you break this up? Looks arbitrarily slapped together and arbitrarily separated. Might be better to sort by the manufacturer, but in any case, this will be obsoleted by a real table. And we'll ponder the peripherals then.

Reply:
[date](https://en.wikipedia.org/wiki/Third_generation_of_video_game_consoles), looks weird because a bunch of cores are "missing"

Looking at it as-is, the peripherals doesn't really make sense. In the table earlier there were a bunch of "?" cells where the wiki was missing info. Removed for now.

Reply:
rose-tinted glasses I guess. now grouped better, hopefully, and sorted alphabetically

Reply:
I mean **just** alphabetic order. Like in your table.

Reply:
done, I think it looks weird now

Reply:
Perfect IMO.